runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkPipelineOptions.java

```diff
@@ -159,6 +159,15 @@
 
   void setEnableMetrics(Boolean enableMetrics);
 
+  @Description(
+      "By default, uses Flink accumulators to store the metrics which allows to query metrics from the PipelineResult. "
+          + "If set to true, metrics will still be reported but can't be queried via PipelineResult. "
+          + "This saves network and memory.")
+  @Default.Boolean(false)
```

Comment:
Nit: I prefer positive flags to avoid the double negative `disabled=false`. Maybe instead something like `useMetricAccumulator` as you did in the tests?

Reply:
Unfortunately, we can't set true-defaulting options to false. That's why I inverted the flag's name. See https://issues.apache.org/jira/browse/BEAM-8959

Reply:
I find it important that flags are named to express their semantics. In this case, the flag's purpose is to turn something off, and there are many similar examples where we would use "skip", "no" etc.

There should not be a case where `disabled=false` needs to be specified by the user. 
But maybe flags should not have default values, since they are fully described by name.

That would also avoid the options to be included with their defaults during job submission, even when the user has not specified them.